includes/request.js

```diff
@@ -37,22 +37,16 @@ function errorHelper(err) {
     }
     debug('Applying response error message with status, url, and method');
     // Override the status text with an improved message
-    let errorMsg = `${err.response.status} ${err.response.statusText || ''}: ` +
-    `${method} ${requestUrl}`;
+    let errorMsg = `${err.response.status} ${method} ${requestUrl}`;
     if (err.response.data) {
       debug('Found response data');
       // Check if we have a JSON response and try to get the error/reason
       if (err.response.headers['content-type'] === 'application/json') {
         debug('Response data is JSON');
-        // Append the error/reason if available
-        if (err.response.data.error) {
-          debug('Augmenting error message with error property');
-          // Override the status text with our more complete message
-          errorMsg += ` - Error: ${err.response.data.error}`;
-          if (err.response.data.reason) {
-            debug('Augmenting error message with reason property');
-            errorMsg += `, Reason: ${err.response.data.reason}`;
-          }
+        // Append the 'errors' message if available
+        if (err.response.data.errors && err.response.data.errors.length > 0) {
+          const originalError = err.response.data.errors[0];
+          errorMsg += ` - Error: ${originalError.message}`;
         }
       } else {
         errorMsg += err.response.data;
```

Comment:
suggest we move the block
```
      // Set a new message for use by the node-sdk-core
      // We use the errors array because it gets processed
      // ahead of all other service errors.
      err.response.data.errors = [{ message: errorMsg }];
```
inside this `else`  so we can accommodate the other suggestion and avoid overwriting the entire array in the other case.